src/CacheInterface.php

```diff
@@ -2,12 +2,22 @@
 
 namespace React\Cache;
 
+use React\Promise\PromiseInterface;
+
 interface CacheInterface
 {
-    // @return React\Promise\PromiseInterface
+    /**
+     * @return PromiseInterface
+     */
     public function get($key);
 
+    /**
+     * @return PromiseInterface
+     */
     public function set($key, $value);
 
+    /**
+     * @return PromiseInterface
```

Comment:
Maybe we could add here what was expect and/or allow to be resolved through the promise. Is it just a simple bool or do we allow implementer specific information to be returned?